src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1201,6 +1201,44 @@ paths:
         default:
             $ref: '#/components/responses/defaultResponse'
 
+  /jobs/{id}/history:
+    parameters:
+      - name: id
+        required: true
+        description: Id of the upload
```

Comment:
The `id` parameter in the path should be related to the path itself. Therefore, the `id` parameter can only be job ID here and not any other id.

Reply:
Also, can you please explain how this endpoint is going to be different from `GET /jobs?upload={id}`?

Reply:
> The `id` parameter in the path should be related to the path itself. Therefore, the `id` parameter can only be job ID here and not any other id.

Actually all these jobs are related to a particular upload ID, but If that's the case, should I change it to `/uploads/{id}/history`?

Reply:
> Also, can you please explain how this endpoint is going to be different from `GET /jobs?upload={id}`?

Yes sure! so this API provides all the components that is required to view the stats in the `show jobs` page
<img width="1422" alt="Screenshot 2022-09-05 at 11 14 39 AM" src="https://user-images.githubusercontent.com/71918441/188368804-2faf261d-4f88-4ce3-b9d5-498fb35f7123.png">

On the other hand `GET jobs?upload={id}` didn't provide all these info
```
[{
        "id": 59,
        "name": "Nirjas-master.zip",
        "queueDate": "2022-09-04 22:22:40.452792+05:30",
        "uploadId": "3",
        "userId": "3",
        "groupId": "3",
        "eta": 0,
        "status": "Failed"
    }, ....]
```
The above response isn't enough to get all the required info in the show jobs page. 
Moreover `GET jobs?upload={id}` only provides the jobs scheduled, It doesn't give relation between what are the sub jobs queued for a particular job. Also we do not get information like `items processed per second`, `type of job`,  `report download information`,  `job dependency` etc.

Reply:
> > The `id` parameter in the path should be related to the path itself. Therefore, the `id` parameter can only be job ID here and not any other id.
> 
> Actually all these jobs are related to a particular upload ID, but If that's the case, should I change it to `/uploads/{id}/history`?

@GMishx Changing the route to `/jobs/history?upload={id}`. Make more sense now.